src/types.ts

```diff
@@ -206,6 +206,7 @@ export type Invoice = {
   payer_pubkey?: string;
   payment_hash: string;
   payment_request: string;
+  preimage: string;
```

Comment:
it might also be null

Reply:
what case is it null right now? these are only settled transactions, right?

But in the future they might not be all settled. Fix here: https://github.com/getAlby/js-sdk/pull/136

Reply:
true the API right now only returns settled invoices.